lib/msf/core/db_manager/import/gpp.rb

```diff
@@ -0,0 +1,41 @@
+require 'rex/parser/group_policy_preferences'
+
+module Msf::DBManager::Import::GPP
```

Comment:
I'm leaving the name as `GPP` for future expansion... and I'm more liable to screw up the rename now.